docs/publish.md

```diff
@@ -0,0 +1,29 @@
+# Publish your app
```

Comment:
add in intro sentence describing this page

Reply:
```suggestion
# Publish your app

This page describes how to setup your CI/CD pipeline to upload your app to App Store Connect and Play Console.
```